src/components/InfoViewer/schemaOverview/CDCStreamOverview.tsx

```diff
@@ -0,0 +1,61 @@
+import type {TEvDescribeSchemeResult, TCdcStreamDescription, TDirEntry} from '../../../types/api/schema';
```

Comment:
this component is the same as `CDCStreamInfo` above
this is a coincidence, not a rule, they should remain separate